CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 4.1.1
+
+- Add support for APS alternative environment variables `APS_CLIENT_ID` and `APS_CLIENT_SECRET`
+
```

Comment:
could you also state that `FORGE_CLIENT_ID` and `FORGE_CLIENT_SECRET` environment variables are not encouraged to use anymore